redhat-ipaas-dev-single-tenant.yml

```diff
@@ -95,6 +95,10 @@ parameters:
   name: POSTGRESQL_VOLUME_CAPACITY
   required: true
   value: 1Gi
+- description: Ignore SSL errors in proxy containers
+  displayName: Insecure skip verify
+  name: INSECURE_SKIP_VERIFY
+  value: 'true'
```

Comment:
You're right, `false` is default. We can keep those test related params to default "restricted" values and configure them accordingly prior to test execution. 
@jludvice could you pls change this?